.devcontainer/development/Dockerfile

```
@@ -118,10 +118,50 @@ RUN /usr/local/bin/apt-install pipx && \
+exit 0
+EOF
+
+# Create the capture file writable by the Postfix pipe user ('nobody').
+RUN touch /var/log/carlos-mail-capture.eml && \
+    chown nobody:nogroup /var/log/carlos-mail-capture.eml && \
+    chmod 0660 /var/log/carlos-mail-capture.eml
+
+# Register a custom pipe transport that streams every accepted message to the
+# capture script. ${sender}/${recipient} pass envelope metadata; the raw
+# message body arrives on stdin.
+RUN printf '%s\n%s\n' \
```

Hadolint (2.14.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)